R/correctionsataglance-data.R

```diff
@@ -203,7 +203,7 @@ parseCorrThresholds <- function(reportObject, timezone){
       type <- rep(threshold[['type']], times=nrow(periods))
       startDates <- periods[['startTime']]
       endDates <- periods[['endTime']]
-      value <- periods[['referenceValue']]
+      value <- if(isEmptyOrBlank(periods[['referenceValue']])){NA} else{periods[['referenceValue']]}
```

Comment:
I'd defer to Lindsay, not sure if the ifelse is a must have but if the if / else works... :shipit: